src/_fetch/_headers.js

```diff
@@ -5,16 +5,27 @@ const makeHeaders = function(options) {
     options['Api-User-Agent'] ||
     'User of the wtf_wikipedia library'
 
+  let origin
+  if (options.noOrigin) {
+    origin = ''
```

Comment:
It will now send emty string as the orgin header. Is that what you want to have happen?

otherwise set it to undefined